connector_test.go

```diff
@@ -115,7 +174,7 @@ var _ = Describe("Connector", func() {
 				objType := ""
 				ref := "fixedaddress/ZG5zLmJpbmRfY25h:12.0.10.1/external"
 				returnFields := []string{}
-				forcedProxy := false
+				var forcedProxy bool
```

Comment:
Why this change? The original `forcedProxy := false` is much clearer.

(There are other instances of this change too).